Dockerfile

```diff
@@ -28,9 +28,14 @@ USER rsk
 WORKDIR /var/lib/rsk
 COPY --from=build --chown=rsk:rsk /home/rsk/rsk.jar ./
 
-ENTRYPOINT ["java", \
-    "-Drpc.providers.web.http.bind_address=0.0.0.0", \
-    "-Drpc.providers.web.http.hosts.0=localhost", \
-    "-Drpc.providers.web.http.hosts.1=127.0.0.1", \
-    "-Drpc.providers.web.http.hosts.2=::1", \
-    "-cp", "rsk.jar", "co.rsk.Start"]
+ENV DEFAULT_JVM_OPTS="-Xms3G -Xmx5G"
+ENV RSKJ_SYS_PROPS="-Drpc.providers.web.http.bind_address=0.0.0.0 -Drpc.providers.web.http.hosts.0=localhost -Drpc.providers.web.http.hosts.1=127.0.0.1 -Drpc.providers.web.http.hosts.2=::1"
+ENV RSKJ_CLASS=co.rsk.Start
+ENV RSKJ_OPTS=""
+
+ENTRYPOINT java \
+    $DEFAULT_JVM_OPTS \
+    $RSKJ_SYS_PROPS \
+    -cp rsk.jar \
+    $RSKJ_CLASS \
+    $RSKJ_OPTS
```

Comment:
one problem I've noticed with this setup is that it doesn't allow to specify a CMD/extra flags, eg. the following command ignores `--reset` flag:
```bash
docker run -d --rm -e RSKJ_OPTS="--regtest" rskj-node --reset
```
It seems that the following trick allows to work that around:
```
ENTRYPOINT ["/bin/sh", "-c", "java $DEFAULT_JVM_OPTS $RSKJ_SYS_PROPS -cp rsk.jar $RSKJ_CLASS $RSKJ_OPTS \"${@}\"", "--"]
```
Let me know what you think on that

Reply:
Done